modules/system/defaults/hitoolbox.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, ... }:
+
+with lib;
```

Comment:
```suggestion
```

Reply:
It's required for mkOption and types so can't remove, unless you mean you prefer `lib.mkOption` and `lib.types`. Will leave out for now.

Reply:
Yeah, can you change them to `lib.mkOption` and `lib.types`?

Reply:
Changed the three uses to lib.*